include/io/Display.hpp

```diff
@@ -30,10 +35,11 @@ namespace chip8::io
                     std::size_t col_;
             };
 
-            PixelColumn operator[](std::size_t row)
+            PixelColumn operator[](std::size_t col)
```

Comment:
Having operator[] return a column is fine, since that allows to chain two of them together. I think it would be nice to have iterators too. You are using an stl container internally, so you got iterators for free. You only need to expose them to users (maybe differentiating between columns iterators and single pixel iterators).

Reply:
Also, shouldn't this return a reference to PixelColumn?

Reply:
How would you expose the iterators? 
Matrix[x][y] would return a Pixel&, right? I'm not sure how to expose them. @Thordreck

Reply:
Check new implementation attempt